lib/challenge.js

```diff
@@ -20,6 +20,23 @@
 
 // Answer Starts Here
 
+'Calendar'
+Decription: Our app keeps track of events for day.
+1) Every Calendar has:
+a) Dates
+b) Hour markers
+c) Months
+
+2) Every calendar can:
+a) Add events
+b) Alert before an event
+c) Repeat events at specified intervals
+
+3) Evey calendar has an event
+a) Block off time
+b) Alert before event
```

Comment:
The goal here was to break up the big abstraction ("a calendar app") into three smaller abstractions. Your one abstraction is well-formed and likely helpful to designing your program, so this gets 1/2.